spec/index.bs

```diff
@@ -736,13 +736,26 @@ For the duration of each scope, the set of [=internal usage=] flags of any given
 A [=subresource=] may transition to new usages at the boundaries between [=usage scope=]s.
 
 This specification defines the following [=usage scopes=]:
-  1. an individual command on a {{GPUCommandEncoder}}, such as {{GPUCommandEncoder/copyBufferToTexture()|GPUCommandEncoder.copyBufferToTexture}}.
-  2. an individual command on a {{GPUComputePassEncoder}}, such as {{GPUProgrammablePassEncoder/setBindGroup(index, bindGroup, dynamicOffsets)|GPUProgrammablePassEncoder.setBindGroup}}.
-  3. the whole {{GPURenderPassEncoder}}.
+
+- Outside of a pass (in {{GPUCommandEncoder}}), each command is one usage scope
+    (e.g. {{GPUCommandEncoder/copyBufferToTexture()}}).
+- In a compute pass, each command is one usage scope.
+    A subresource is "used" in the usage scope if it's accessible by the command.
+    For example, in {{GPUComputePassEncoder/dispatch()}},
+    every subresource in every currently bound bind group is "used" in the usage scope.
+    (Compute pass state-setting commands are trivially-valid usage scopes; see below.)
+- One render pass is one usage scope.
+    A subresource is "used" in the usage scope if it's referenced by any command,
+    including state setting commands
+    (like {{GPUProgrammablePassEncoder/setBindGroup(index, bindGroup, dynamicOffsets)}}).
+
+Issue: The above should probably talk about [=GPU commands=]. But we don't have a way to
+reference specific GPU commands (like dispatch) yet.
+
+Resources that are used in state-setting calls in passes are always added as used resources in the containing [=usage scope=].
```

Comment:
This is misleading. It's written as if the usage scopes are not intersecting, and there is only one of them to get the usages of resources in state-setting calls. But in compute we have multiple dispatches with their own usage scopes.